test/Main.purs

```diff
@@ -62,6 +66,12 @@ route =
   bazRoute =
     segment / rest
 
+  quxRoute =
+    variant
+      # _list %= pure unit
```

Comment:
`inj @"list"` and `Root` will clash bc they both match to `/`

Reply:
fixed it in https://github.com/natefaubion/purescript-routing-duplex/pull/38 by moving to https://github.com/natefaubion/purescript-routing-duplex/pull/38/commits/7843c0126ad97bcf6b330300d770d5607fa950d6#diff-bc7f1b844379519c7cd4d7b2a2275dade0ab1adcd22efb61268c8a9d6044d6bcR26